src/snovault/nginx-dev.conf

```diff
@@ -11,6 +11,12 @@ http {
         server 127.0.0.1:6543;
         keepalive 10;
     }
+
+    map $http_x_forwarded_proto $forwarded_proto {
```

Comment:
if I follow this it is just saying to use http/https based on value of http_x_forwarded_proto or scheme.  Although I am not sure how these are set.

Reply:
Yes. It's using the X-Forwarded-Proto header if it exists and the scheme if it doesn't. That header is set by a downstream proxy server (in my case the Google Cloud Integrated Authentication Proxy I use for authentication.) I needed this so I could test my authentication setup locally.